modules/ding_event/ding_event.views_default.inc

```diff
@@ -581,7 +581,7 @@ function ding_event_views_default_views() {
   $handler->display->display_options['filters']['combine']['id'] = 'combine';
   $handler->display->display_options['filters']['combine']['table'] = 'views';
   $handler->display->display_options['filters']['combine']['field'] = 'combine';
-  $handler->display->display_options['filters']['combine']['operator'] = 'allwords';
+  $handler->display->display_options['filters']['combine']['operator'] = 'contains';
```

Comment:
https://inlead.atlassian.net/browse/EASYOPAC-683

Reply:
I think you linked the wrong issue?

Reply:
https://github.com/easySuite/ding2/commit/84c1dc4dade9e4eff766a69a5db6fda0cda63288#diff-910ba0361c20fb0a154046bcf13046d70e01eae0bd1ff940848ec844352625bbR581

Reply:
Well I guess its wrong.

Reply:
How do we proceed here?